_data/shared_hosts.yml

```diff
@@ -143,6 +143,14 @@
   php54: ??
   default: 5.4.??
 
+- name: "Kinghost"
```

Comment:
I'm not longer able to add any hosts with `??` in there, as it messes up my SemVer-based sorting. Can you find out the patch level versions of this so I can merge it?